src/main/modules/Execution.js

```diff
@@ -3,90 +3,83 @@ import { Validation } from './Validation';
 var actualActor = 0;
 var validationExectute = new Validation;
 export class Execution {
-    constructor(parserActionsArray, parserActorsArray, scenes) {
-        this.parserActionsArray = parserActionsArray;
-        this.parserActorsArray = parserActorsArray;
+    constructor(actionsArray, actorsArray, scenes) {
+        this. actionsArray =  actionsArray;
```

Comment:
The space after the dot is intentional?

Reply:
Have the same doubt. Also it's the same in actors

Reply:
Ups, that space appears when i was doing the renaming process, and i think i forgot to delete right there, but i tried to delete that space in every place that actionsArray is, thank you for the observation.

1. It's not intentional, just a renaming process space
2. I'm going to change it :D 
3. this space doesn't affect the functionality.
4. Good observation.